x-pack/plugins/apm/public/components/shared/Stacktrace/LibraryFrames.tsx

```diff
@@ -20,6 +20,26 @@ const LibraryFrameToggle = styled.div`
   user-select: none;
 `;
 
+interface LibraryStackFrameProps {
+  codeLanguage?: string;
+  stackframe: Stackframe;
+}
+
+const LibraryStackFrame: React.SFC<LibraryStackFrameProps> = ({
```

Comment:
This method is called `LibraryStackFrame` and the other `LibraryFrames`. If the only diff is singular/plural should the be more consistent eg. `LibraryStackFrame` and `LibraryStacokFrames`?